src/gitops/components/shared/DataView/gitOpsDataViewPagination.test.ts

```
@@ -83,4 +88,94 @@ describe('getGitOpsPaginationResetKey', () => {
+    const roles = [
+      { name: 'ci-role' },
+      { name: 'read-only' },
+      { name: 'admin' },
+      { name: 'developer' },
+    ].sort((a, b) => a.name.localeCompare(b.name));
+
+    expect(paginateItems(roles, 1, 2).map((role) => role.name)).toEqual(['admin', 'ci-role']);
+    expect(paginateItems(roles, 2, 2).map((role) => role.name)).toEqual([
+      'developer',
+      'read-only',
+    ]);
```

ESLint

[error] Replace `⏎······'developer',⏎······'read-only',⏎····` with `'developer',·'read-only'`

(prettier/prettier)